examples/05_stable_diffusion/src/compile_lib/compile_unet.py

```diff
@@ -85,4 +86,19 @@ def compile_unet(
     target = detect_target(
         use_fp16_acc=use_fp16_acc, convert_conv_to_gemm=convert_conv_to_gemm
     )
-    compile_model(Y, target, "./tmp", "UNet2DConditionModel", constants=params_ait)
+    """
+        Set the OS environment variable AITEMPLATE_WORK_DIR to point to an absolute path to a directory which 
+        will be used to save the AIT compiled model artifacts. Make sure the OS user running this script has read and write 
+        permissions to this directory. By default, the artifacts will be saved under tmp/ folder of the 
+        current working directory. 
+    """
+
+    env_name = "AITEMPLATE_WORK_DIR"
+    try:
+        if os.environ[env_name]:
+            workdir = os.environ[env_name]
+            print("The value of", env_name, " is ", workdir)
+    except KeyError:
+        workdir = "tmp/"
+        print("The value of", env_name, " is ", workdir)   
+    compile_model(Y, target,workdir, "UNet2DConditionModel", constants=params_ait)
```

Comment:
Ditto about 1) indentation, 2) usage of try/except, 3) spaces between args, 4) string formatting.

Reply:
Done